src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2575,10 +2584,10 @@ export class PostgresStorageAdapter implements StorageAdapter {
         ) {
           // Cast the error into the proper parse error
           throw new Parse.Error(
-            Parse.Error.DUPLICATE_VALUE,
-            'A duplicate value for a field with unique values was provided'
-          );
-        } else {
+			Parse.Error.DUPLICATE_VALUE,
+			'A duplicate value for a field with unique values was provided'
+		  );
+		} else {
```

Comment:
Use 2 spaces instead of tabs

Reply:
It's the same thing. I tried setting tab indentation to two spaces and it still fails
```

✖ eslint found some errors. Please fix them and try committing again.

/Users/siddharthramesh/Desktop/code/open-source/parse-server/src/Adapters/Storage/Postgres/PostgresStorageAdapter.js
  2587:1  error  Expected indentation of 8 spaces but found 10  indent
  2588:1  error  Expected indentation of 6 spaces but found 8   indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

Can you merge it and again re-commit with the right formatting?

Reply:
@dplewis @cbaker6 
```npm run lint --fix``` return no error:
```

But running git commit without no-verify fails

Reply:
I ran into the same issue you did. Its weird, I know of a fix but its not recommended.